src/Configurator/EnvConfigurator.php

```diff
@@ -18,7 +18,7 @@
  */
 class EnvConfigurator extends AbstractConfigurator
 {
-    public function configure(Recipe $recipe, $vars)
+    public function configure(Recipe $recipe, $vars, array $contextParameters = [])
```

Comment:
maybe just "$options" or "$parameters" (should match getParameters/getOptions on UpdateEvent)

Reply:
Depending on the configurators, the previous argument can be called `$config` or `$parameters` , so there might be potential name conflicts :confused: